package.json

```diff
@@ -5,7 +5,7 @@
   "main": "src/index.js",
   "scripts": {
     "build": "ncc build src/index.js --license licenses.txt -o dist/main",
-    "test": "npx jest",
+    "test": "npx jest --forceExit",
```

Comment:
Note: [This sleep here](https://github.com/saucelabs/saucectl-run-action/blob/4e1ffcd4918f5dd3844012dcefab7b34497bdbb7/src/helpers.js#L40-L40) is causing jest to fail, even though it exits just fine, but will do so with an error code of 1. A forced exit resolves this problem. May or may not be the best solution, but didn't want to go even deeper into the rabbit hole at this point of time.